test/rockpaperscissors.js

```diff
@@ -0,0 +1,772 @@
+            });
+
+        it("Value sent that exceeds the stake, is sent to the player's balance", async () => {
+            gameKey = await rcp.createPlayerOneMoveHash(playerOne, playerOneSecretBytes32, PAPER);
+
+            const initPlayerOneOwed = await rcp.balances(playerOne);
+            assert.strictEqual(initPlayerOneOwed.toString(10), "0");
+
+            // 7 - 5 = 2 is added to the balance
+            await rcp.movePlayerOne(gameKey, toBN(5), {from: playerOne, value: toBN(7)});
+
+            const playerOneOwed1 = await rcp.balances(playerOne);
+            assert.strictEqual(playerOneOwed1.toString(10), "2");
+
+            // 10 - 5 = 5 is added to the balance
+            gameKey = await rcp.createPlayerOneMoveHash(playerOne, playerOneSecretBytes32, ROCK);
+            await rcp.movePlayerOne(gameKey, toBN(5), {from: playerOne, value: toBN(10)});
+
+            // Total balance is now 5 + 2
+            const playerOneOwed2 = await rcp.balances(playerOne);
+            assert.strictEqual(playerOneOwed2.toString(10), "7");
+
+        });
+
+        it("Player two hashes their address, game key, secret and move", async () => {
+            gameKey = await rcp.createPlayerOneMoveHash(playerOne, playerOneSecretBytes32, ROCK);
+            const playerTwoMoveHash = await rcp.createPlayerTwoMoveHash(playerTwo, gameKey, playerTwoSecretBytes32, SCISSORS);
+            const expectedMoveHash =  await soliditySha3(playerTwo, gameKey, playerTwoSecretBytes32, SCISSORS);
+            assert.strictEqual(expectedMoveHash, playerTwoMoveHash);
+        });
+
+        it("Player two moves", async () => {
+
+            await rcp.movePlayerOne(gameKey, toBN(10), {from: playerOne, value: toBN(10)});
+            const txObj = await rcp.movePlayerTwo(gameKey, playerTwoMoveHash, {from: playerTwo, value: toBN(10)});
+
+            const timestamp = await getBlockTimeStamp(txObj);
+            const expiryDate = timestamp + DAY_24_HOUR_IN_SECS;
+
+            await truffleAssert.eventEmitted(txObj, 'PlayerTwoMoves', (ev) => {
+
+                return  ev.player === playerTwo &&
+                        ev.amount.toString(10) === "10" &&
+                        ev.gameKey === gameKey &&
+                        ev.moveHash === playerTwoMoveHash &&
+                        ev.expiryDate.toString(10) === expiryDate.toString(10)
+            },'PlayerTwoMoves event is emitted');
+
+            const game = await rcp.games(gameKey);
+
+            assert.strictEqual(game.playerTwo, playerTwo);
+            assert.strictEqual(game.playerTwoMoveHash, playerTwoMoveHash);
+            assert.strictEqual(game.step.toString(10), PLAYER_TWO_MOVE);
+        });
+    });
+
+    describe("Player reveals", async () => {
+
+        it("Player one reveals", async () => {
+
+            gameKey = await rcp.createPlayerOneMoveHash(playerOne, playerOneSecretBytes32, SCISSORS);
+            await rcp.movePlayerOne(gameKey, toBN(10), {from: playerOne, value: toBN(10)});
```

Comment:
Would passing `"10"` work?

Reply:
Yes. I have been guilty of too much cut and paste.